src/pip/_internal/utils/misc.py

```diff
@@ -948,6 +948,10 @@ def call_subprocess(
     return ''.join(all_output)
 
 
+def write_output(*args):
```

Comment:
I would make this signature `write_output(msg, *args)` (using the same argument name as the `logging` module) to make clear that the first argument is distinguished and has special meaning.